api/libproject/devstart.rb

```diff
@@ -1124,39 +1124,6 @@ def run_drop_cdr_db()
   :fn => ->(*args) { run_cloud_data_migrations("run-cloud-data-migrations", args) }
 })
 
-def write_db_creds_file(project, cdr_db_name, root_password, workbench_password, readonly_password)
```

Comment:
The source of truth for this information is the credentials file itself in the GCS bucket. If it needs updating, download, edit, upload.